core/blockchain/impl/weight_calculator_impl.cpp

```diff
@@ -5,13 +5,46 @@
 
 #include "blockchain/impl/weight_calculator_impl.hpp"
 
+#include "vm/actor/builtin/storage_power/storage_power_actor_state.hpp"
+#include "vm/state/impl/state_tree_impl.hpp"
+
+/* LCOV_EXCL_START */
```

Comment:
This comment doesn't take an effect for codecov (see https://codecov.io/gh/filecoin-project/cpp-filecoin/pull/141/src/core/blockchain/impl/weight_calculator_impl.cpp?before=core/blockchain/impl/weight_calculator_impl.cpp).